src/server/lib/logger.ts

```diff
@@ -9,8 +9,21 @@ export const logger: Logger = {
   log(level: LogLevel, message: string) {
     winstonLogger.log(level, message);
   },
-  error(message: string) {
-    logger.log(LogLevel.ERROR, message);
+  // errors can be anything
```

Comment:
I feel that this could be addressed, but not in this PR, might be worth something to think about as a team.

Reply:
This PR needs a mechanism to log the stack traces if they exist. I'm happy to implement an alternative or have a chat about it.

Reply:
Oh this is out of scope of the PR, but I wanted to flag that you had to do this in order for this PR to work and that we can see, as a separate investigation if there's a nicer way we can deal with error shapes.

Reply:
👍  I misunderstood